dashboard/test/models/ability_test.rb

```diff
@@ -29,70 +47,302 @@ class AbilityTest < ActiveSupport::TestCase
+    assert ability.can?(:read, @public_universal_instructor_to_teacher_script_level)
+    assert ability.can?(:read, @public_plc_reviewer_to_facilitator_script_level)
+    assert ability.can?(:read, @public_teacher_to_student_script_level, {login_required: "true"})
+    assert ability.can?(:read, @login_required_script_level)
 
+    assert ability.can?(:read, @pilot_course_script_level)
+    assert ability.can?(:read, @pl_pilot_course_script_level)
+  end
+
+  test "as pilot plc reviewer" do
+    ability = Ability.new(create(:plc_reviewer, pilot_experiment: 'my-experiment'))
+    assert ability.can?(:read, Game)
+    assert ability.can?(:read, Level)
+    assert ability.can?(:read, Activity)
     refute ability.can?(:destroy, Game)
     refute ability.can?(:destroy, Level)
     refute ability.can?(:destroy, Activity)
+    assert ability.can?(:read, Section)
+    assert ability.can?(:read, Script.find_by_name('ECSPD'))
+    assert ability.can?(:read, Script.find_by_name('flappy'))
+
+    assert ability.can?(:read, Script.find_by_name('ECSPD'))
+    assert ability.can?(:read, Script.find_by_name('flappy'))
+
+    assert ability.can?(:read, @public_teacher_to_student_unit)
+    assert ability.can?(:read, @public_facilitator_to_teacher_unit)
+    assert ability.can?(:read, @public_universal_instructor_to_teacher_unit)
+    assert ability.can?(:read, @public_plc_reviewer_to_facilitator_unit)
+
+    assert ability.can?(:read, @pilot_course)
+    assert ability.can?(:read, @pl_pilot_course)
+
+    assert ability.can?(:read, @login_required_migrated_lesson)
+    assert ability.can?(:student_lesson_plan, @login_required_migrated_lesson)
+
+    assert ability.can?(:read, @public_teacher_to_student_script_level)
+    assert ability.can?(:read, @public_facilitator_to_teacher_script_level)
+    assert ability.can?(:read, @public_universal_instructor_to_teacher_script_level)
+    assert ability.can?(:read, @public_plc_reviewer_to_facilitator_script_level)
+    assert ability.can?(:read, @public_teacher_to_student_script_level, {login_required: "true"})
+    assert ability.can?(:read, @login_required_script_level)
 
+    assert ability.can?(:read, @pilot_course_script_level)
+    assert ability.can?(:read, @pl_pilot_course_script_level)
+  end
+
+  test "as guest" do
+    ability = Ability.new(User.new)
+    assert ability.can?(:read, Game)
+    assert ability.can?(:read, Level)
+    assert ability.can?(:read, Activity)
+    refute ability.can?(:destroy, Game)
+    refute ability.can?(:destroy, Level)
+    refute ability.can?(:destroy, Activity)
     refute ability.can?(:read, Section)
+    assert ability.can?(:read, Script.find_by_name('ECSPD'))
+    assert ability.can?(:read, Script.find_by_name('flappy'))
 
     assert ability.can?(:read, Script.find_by_name('ECSPD'))
     assert ability.can?(:read, Script.find_by_name('flappy'))
 
     refute ability.can?(:read, @in_development_script)
```

Comment:
nit: this check isn't in the plc pilot tests but might be worth adding to those as well?

Reply:
I'm not sure I understand what test you are suggesting I add this too. Could you clarify?

Reply:
I see these lines being tested in those test right now.

Reply:
oh weird, maybe the diff hid them. Anyway, this PR is def good to go!